driver/src/main/java/org/neo4j/driver/internal/cluster/Rediscovery.java

```diff
@@ -47,13 +48,11 @@ public Rediscovery( RoutingSettings settings, Clock clock, Logger logger, Cluste
     // Given the current routing table and connection pool, use the connection composition provider to fetch a new
     // cluster composition, which would be used to update the routing table and connection pool
     public ClusterComposition lookupRoutingTable( ConnectionPool connections, RoutingTable routingTable )
-            throws InterruptedException, ServiceUnavailableException
+            throws InterruptedException, ServiceUnavailableException, ProtocolException
```

Comment:
Do we really need to declare runtime exceptions `ServiceUnavailableException` and `ProtocolException` here?